srv/salt/ceph/configuration/files/ceph.conf-shared.j2

```diff
@@ -8,6 +8,7 @@ auth_client_required = cephx
 filestore_xattr_use_omap = true
 public_network = {{ salt['pillar.get']('public_network') }}
 cluster_network = {{ salt['pillar.get']('cluster_network') }}
+rbd default features = 3
```

Comment:
In SES5 we already support textual values for the RBD features, like this:
`rbd default features = layering,striping`

But if this configuration needs to work with versions <SES5 then it looks good to me